docs/make.bat

```diff
@@ -1,3 +1,18 @@
+# Copyright 2018 Intel Corporation

```

Comment:
Much of this file is auto-generated by the sphinx-quickstart tool. I don't think Intel should claim copyright.

We should probably remove make.bat completely (windows users should build in docker).